### ISSUE TITLE
Weird error calculating f1 metric

### ISSUE THREAD

**Issue by nanounanue** [dssg/triage#226]
```
13:04:55
ERROR:root:Child error: Traceback (most recent call last):

13:04:55
File "/usr/local/lib/python3.6/site-packages/triage/experiments/multicore.py", line 276, in test_and_evaluate

13:04:55
example_frequency=split_def['example_frequency']

13:04:55
File "/usr/local/lib/python3.6/site-packages/catwalk/evaluation.py", line 225, in evaluate

13:04:55
labels_sorted.tolist(),

13:04:55
File "/usr/local/lib/python3.6/site-packages/catwalk/evaluation.py", line 145, in _generate_evaluations

13:04:55
parameter_combination

13:04:55
File "/usr/local/lib/python3.6/site-packages/catwalk/metrics.py", line 46, in __call__

13:04:55
return self.function(*params, **kwparams)

13:04:55
File "/usr/local/lib/python3.6/site-packages/catwalk/metrics.py", line 69, in f1

13:04:55
return metrics.f1_score(labels, predictions_binary, **parameters)

13:04:55
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 692, in f1_score

13:04:55
sample_weight=sample_weight)

13:04:55
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 806, in fbeta_score

13:04:55
sample_weight=sample_weight)

13:04:55
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 1003, in precision_recall_fscore_support

13:04:55
y_type, y_true, y_pred = _check_targets(y_true, y_pred)

13:04:55
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 82, in _check_targets

13:04:55
"".format(type_true, type_pred))

13:04:55
ValueError: Can't handle mix of continuous and binary

13:05:07
WARNING:root:1year

13:05:09
ERROR:root:Child error: Traceback (most recent call last):

13:05:09
File "/usr/local/lib/python3.6/site-packages/triage/experiments/multicore.py", line 276, in test_and_evaluate

13:05:09
example_frequency=split_def['example_frequency']

13:05:09
File "/usr/local/lib/python3.6/site-packages/catwalk/evaluation.py", line 225, in evaluate

13:05:09
labels_sorted.tolist(),

13:05:09
File "/usr/local/lib/python3.6/site-packages/catwalk/evaluation.py", line 145, in _generate_evaluations

13:05:09
parameter_combination

13:05:09
File "/usr/local/lib/python3.6/site-packages/catwalk/metrics.py", line 46, in __call__

13:05:09
return self.function(*params, **kwparams)

13:05:09
File "/usr/local/lib/python3.6/site-packages/catwalk/metrics.py", line 69, in f1

13:05:09
return metrics.f1_score(labels, predictions_binary, **parameters)

13:05:09
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 692, in f1_score

13:05:09
sample_weight=sample_weight)

13:05:09
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 806, in fbeta_score

13:05:09
sample_weight=sample_weight)

13:05:09
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 1003, in precision_recall_fscore_support

13:05:09
y_type, y_true, y_pred = _check_targets(y_true, y_pred)

13:05:09
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 82, in _check_targets

13:05:09
"".format(type_true, type_pred))

13:05:09
ValueError: Can't handle mix of continuous and binary

13:05:23
WARNING:root:1year

13:05:26
ERROR:root:Child error: Traceback (most recent call last):

13:05:26
File "/usr/local/lib/python3.6/site-packages/triage/experiments/multicore.py", line 276, in test_and_evaluate

13:05:26
example_frequency=split_def['example_frequency']

13:05:26
File "/usr/local/lib/python3.6/site-packages/catwalk/evaluation.py", line 225, in evaluate

13:05:26
labels_sorted.tolist(),

13:05:26
File "/usr/local/lib/python3.6/site-packages/catwalk/evaluation.py", line 145, in _generate_evaluations

13:05:26
parameter_combination

13:05:26
File "/usr/local/lib/python3.6/site-packages/catwalk/metrics.py", line 46, in __call__

13:05:26
return self.function(*params, **kwparams)

13:05:26
File "/usr/local/lib/python3.6/site-packages/catwalk/metrics.py", line 69, in f1

13:05:26
return metrics.f1_score(labels, predictions_binary, **parameters)

13:05:26
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 692, in f1_score

13:05:26
sample_weight=sample_weight)

13:05:26
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 806, in fbeta_score

13:05:26
sample_weight=sample_weight)

13:05:26
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 1003, in precision_recall_fscore_support

13:05:26
y_type, y_true, y_pred = _check_targets(y_true, y_pred)

13:05:26
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 82, in _check_targets

13:05:26
"".format(type_true, type_pred))

13:05:26
ValueError: Can't handle mix of continuous and binary

13:05:41
WARNING:root:1year

13:05:44
ERROR:root:Child error: Traceback (most recent call last):

13:05:44
File "/usr/local/lib/python3.6/site-packages/triage/experiments/multicore.py", line 276, in test_and_evaluate

13:05:44
example_frequency=split_def['example_frequency']

13:05:44
File "/usr/local/lib/python3.6/site-packages/catwalk/evaluation.py", line 225, in evaluate

13:05:44
labels_sorted.tolist(),

13:05:44
File "/usr/local/lib/python3.6/site-packages/catwalk/evaluation.py", line 145, in _generate_evaluations

13:05:44
parameter_combination

13:05:44
File "/usr/local/lib/python3.6/site-packages/catwalk/metrics.py", line 46, in __call__

13:05:44
return self.function(*params, **kwparams)

13:05:44
File "/usr/local/lib/python3.6/site-packages/catwalk/metrics.py", line 69, in f1

13:05:44
return metrics.f1_score(labels, predictions_binary, **parameters)

13:05:44
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 692, in f1_score

13:05:44
sample_weight=sample_weight)

13:05:44
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 806, in fbeta_score

13:05:44
sample_weight=sample_weight)

13:05:44
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 1003, in precision_recall_fscore_support

13:05:44
y_type, y_true, y_pred = _check_targets(y_true, y_pred)

13:05:44
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 82, in _check_targets

13:05:44
"".format(type_true, type_pred))

13:05:44
ValueError: Can't handle mix of continuous and binary

13:05:59
WARNING:root:1year

13:06:02
ERROR:root:Child error: Traceback (most recent call last):

13:06:02
File "/usr/local/lib/python3.6/site-packages/triage/experiments/multicore.py", line 276, in test_and_evaluate

13:06:02
example_frequency=split_def['example_frequency']

13:06:02
File "/usr/local/lib/python3.6/site-packages/catwalk/evaluation.py", line 225, in evaluate

13:06:02
labels_sorted.tolist(),

13:06:02
File "/usr/local/lib/python3.6/site-packages/catwalk/evaluation.py", line 145, in _generate_evaluations

13:06:02
parameter_combination

13:06:02
File "/usr/local/lib/python3.6/site-packages/catwalk/metrics.py", line 46, in __call__

13:06:02
return self.function(*params, **kwparams)

13:06:02
File "/usr/local/lib/python3.6/site-packages/catwalk/metrics.py", line 69, in f1

13:06:02
return metrics.f1_score(labels, predictions_binary, **parameters)

13:06:02
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 692, in f1_score

13:06:02
sample_weight=sample_weight)

13:06:02
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 806, in fbeta_score

13:06:02
sample_weight=sample_weight)

13:06:02
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 1003, in precision_recall_fscore_support

13:06:02
y_type, y_true, y_pred = _check_targets(y_true, y_pred)

13:06:02
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 82, in _check_targets

13:06:02
"".format(type_true, type_pred))

13:06:02
ValueError: Can't handle mix of continuous and binary

13:06:18
WARNING:root:1year

13:06:21
ERROR:root:Child error: Traceback (most recent call last):

13:06:21
File "/usr/local/lib/python3.6/site-packages/triage/experiments/multicore.py", line 276, in test_and_evaluate

13:06:21
example_frequency=split_def['example_frequency']

13:06:21
File "/usr/local/lib/python3.6/site-packages/catwalk/evaluation.py", line 225, in evaluate

13:06:21
labels_sorted.tolist(),

13:06:21
File "/usr/local/lib/python3.6/site-packages/catwalk/evaluation.py", line 145, in _generate_evaluations

13:06:21
parameter_combination

13:06:21
File "/usr/local/lib/python3.6/site-packages/catwalk/metrics.py", line 46, in __call__

13:06:21
return self.function(*params, **kwparams)

13:06:21
File "/usr/local/lib/python3.6/site-packages/catwalk/metrics.py", line 69, in f1

13:06:21
return metrics.f1_score(labels, predictions_binary, **parameters)

13:06:21
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 692, in f1_score

13:06:21
sample_weight=sample_weight)

13:06:21
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 806, in fbeta_score

13:06:21
sample_weight=sample_weight)

13:06:21
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 1003, in precision_recall_fscore_support

13:06:21
y_type, y_true, y_pred = _check_targets(y_true, y_pred)

13:06:21
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 82, in _check_targets

13:06:21
"".format(type_true, type_pred))

13:06:21
ValueError: Can't handle mix of continuous and binary

13:06:37
WARNING:root:1year

13:06:40
ERROR:root:Child error: Traceback (most recent call last):

13:06:40
File "/usr/local/lib/python3.6/site-packages/triage/experiments/multicore.py", line 276, in test_and_evaluate

13:06:40
example_frequency=split_def['example_frequency']

13:06:40
File "/usr/local/lib/python3.6/site-packages/catwalk/evaluation.py", line 225, in evaluate

13:06:40
labels_sorted.tolist(),

13:06:40
File "/usr/local/lib/python3.6/site-packages/catwalk/evaluation.py", line 145, in _generate_evaluations

13:06:40
parameter_combination

13:06:40
File "/usr/local/lib/python3.6/site-packages/catwalk/metrics.py", line 46, in __call__

13:06:40
return self.function(*params, **kwparams)

13:06:40
File "/usr/local/lib/python3.6/site-packages/catwalk/metrics.py", line 69, in f1

13:06:40
return metrics.f1_score(labels, predictions_binary, **parameters)

13:06:40
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 692, in f1_score

13:06:40
sample_weight=sample_weight)

13:06:40
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 806, in fbeta_score

13:06:40
sample_weight=sample_weight)

13:06:40
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 1003, in precision_recall_fscore_support

13:06:40
y_type, y_true, y_pred = _check_targets(y_true, y_pred)

13:06:40
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/classification.py", line 82, in _check_targets

13:06:40
"".format(type_true, type_pred))

13:06:40
ValueError: Can't handle mix of continuous and binary
```

**Comment by thcrock** [comment]
This is covered by https://github.com/dssg/catwalk/pull/30 which has to be reopened against this repository.

**Comment by thcrock** [comment]
The PR in question did get merged, so we should check to see if this bug is fixed now

**Comment by thcrock** [comment]
Fixed in #373
